lib/components/normal-components/Board.ts

```diff
@@ -67,6 +69,38 @@ export class Board extends Group<typeof boardProps> {
   _drcChecksComplete = false
   _connectedSchematicPortPairs = new Set<string>()
 
+  private _applyBoardAnchorAlignment(
```

Comment:
two parameter rule:  https://github.com/tscircuit/handbook/blob/main/guides/code.md

also the name of this function is wrong, it doesn't have any effects (it doesn't mutate anything) so it shouldn't use the word `apply`, i also don't understand what it's returning but it's not a board alignment. I think this function could be called `computeBoardAlignmentOffset` and maybe moved out of this file (it's important to keep these class files very small), you can turn `Board.ts` into `Board/Board.ts` and have `Board/computeBoardCenterWithAlignment.ts`